metricbeat/module/kafka/broker/manifest.yml

```diff
@@ -11,81 +11,81 @@ input:
         attributes:
           - attr: Value
             field: request.channel.queue.size
-      - mbean: 'kafka.server:name=FailedProduceRequestsPerSec,type=BrokerTopicMetrics'
+      - mbean: 'kafka.server:type=BrokerTopicMetrics,name=FailedProduceRequestsPerSec'
```

Comment:
@gquintana was this change needed?

Reply:
No it's not needed, but
* it makes more readable, as names get vertically aligned (you'll notice there are metric duplication)
* it matches Kafka documentation https://kafka.apache.org/documentation/#monitoring

Reply:
Sounds good, thanks :+1: